crates/e2e/sandbox/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "ink_sandbox"
+version.workspace = true
```

Comment:
this should probably be un-pegged to the workspace version so that drink! can push update independently of ink! releases

Reply:
I don't see any down side doing it. It also does not depend on other local crate defined in the repo so I don't see that as being problematic.

Reply:
I'm on board with giving this its own independent version. Would need to update the release checklist for that.